js/containers/DeployPlan.jsx

```diff
@@ -10,6 +10,17 @@ const Messages = require('../components/Messages.jsx');
 const actions = require('../_actions.js');
 
 function DeployPlan(props) {
+	// don't show the rest of the form if there is an error in the deploy plan
+	if (props.messages.length && props.messages[0].code == 'error') {
```

Comment:
you can check `validation_code` (there's some other code that does this somewhere)